### PR TITLE
fix: Display BG on DCDU & SAI

### PR DESCRIPTION
**Pull request by NathanInnes**
## Summary of Changes

Changed DCDU & SAI backlight to match the other displays
## Screenshots (if necessary)
image

## References

image

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Spawn C&D and check colour when off, and colour when on

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artefact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artefacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by tjburni** [comment]
Category: QA Tester
Name: Category: QA Tester
Name: [Z+1]tjburni#0409
Date of testing: Current date 05.1.2021
Version of the sim: 1.12.13
PR Tested: #2881
Object of test: DCDU Bisplay

Testing technique:tested?
Steps you've made to test the Pull Request:
1. started C&D at Gate
2. started plane
3.observed the in the PR described MCDU Pages
 observed the in the PR described MCDU Pages



Flight notes (If required):

Issues (If Any): none
Overall Rating: Good
Conclusions of the testing:Looks Good. I see no big difference but the backlight matches the other displays now. Thanks for these Improments!

**Comment by St54Kevin** [comment]
> Category: QA Tester
> Name: Category: QA Tester
> Name: [Z+1]tjburni#0409
> Date of testing: Current date 05.1.2021
> Version of the sim: 1.12.13
> PR Tested: #2881
> Object of test: DCDU Bisplay
> 
> Testing technique:tested?
> Steps you've made to test the Pull Request:
> 
> 1. started C&D at Gate
> 2. started plane
>    3.observed the in the PR described MCDU Pages
>    observed the in the PR described MCDU Pages
> 
> Flight notes (If required):
> 
> Issues (If Any): none
> Overall Rating: Good
> Conclusions of the testing:Looks Good. I see no big difference but the backlight matches the other displays now. Thanks for these Improments!

This PR is about the DCDU not the MCDU ;)

**Comment by St54Kevin** [comment]
Category: QA Tester
Name: Category: QA Tester
Name: [Z+1]tjburni#0409
Date of testing: Current date 05.1.2021
Version of the sim: 1.12.13
PR Tested: #2881
Object of test: DCDU Display brightness

Testing technique:
Steps you've made to test the Pull Request:
1. Spawned at Gate C&D
2. Powered airplane up
3. Tested whole range of DCDU brightness and compare to other displays

Flight notes (If required):
1. Tested in daylight and nightlight

Issues (If Any): -
Overall Rating: LGTM
Conclusions of the testing: Works as expected. Brightness seems overall even

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/SAI/A320_Neo_SAI.html [hunk 1]
```diff
@@ -5,11 +5,12 @@
 			<svg width="100%" height="100%" >
 				<defs>
 					<radialGradient id="SAIBacklight" cx="50%" cy="50%" r="75%" >
-						<stop offset="0%" style="stop-color:rgb(13,20,35);stop-opacity:1" />
-						<stop offset="100%" style="stop-color:rgb(13,20,35);stop-opacity:1" />
+						<stop offset="0%" style="stop-color:rgba(4, 4, 5, 1);stop-opacity:1" />
+						<stop offset="100%" style="stop-color:rgba(4, 4, 5, 1);stop-opacity:1" />
```
Comment (pepperoni505) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/SAI/A320_Neo_SAI.html:9]: For consistency, why use RGBA for this? The rest of the backlighting uses RGB AFAIK

**Comment by ChruutvoLuzi** [comment]
Category: QA Tester
Name: ChruutvoLuzi | SD-PB.ch#8902
Date of testing: 5.1.2021
Version of the sim: 1.12.13.0
PR Tested: #2881 
Object of test: DCDU Backlight Fix

Testing technique: Try out
Steps you've made to test the Pull Request:
1. Spawned C&D Gate A44 LSZH
2. Started up the AC
3. tested through different brightness settings on the DCDU

Flight notes (If required): none

Issues (If Any): none
Overall Rating: Good
Conclusions of the testing: works as described, brightness and look is the same

ECAM+DCDU Brightest
Microsoft Flight Simulator Screenshot 2021 01 05 - 17 39 23 16

ECAM+DCDU Lowest
Microsoft Flight Simulator Screenshot 2021 01 05 - 17 39 41 21

**Comment by NathanInnes** [comment]
Completely forgot to mention the SAI in this PR. Note - this has also been changed.

**Review by pepperoni505 — APPROVED** [review]

**Comment by pepperoni505** [comment]
This is a simple change and has been tested, so I'm going to merge

**Pull request merged**